server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -357,6 +357,8 @@ const orderMutations = {
         throw new Unauthorized("You don't have permission to update this order");
       } else if (!order.Subscription.isActive) {
         throw new Error('Order must be active to be updated');
+      } else if (order.status === OrderStatuses.PAUSED) {
+        throw new Error('Paused orders cannot be updated');
```

Comment:
Maybe we'll need to revisit this once we're ready? Updating a payment method should be ok?

Reply:
Absolutely. Here I wanted to prevent people re-activating their orders while the collective is still under OCF, until we build the proper solution for it.